diff -ru a/hack/lib/golang.sh b/hack/lib/golang.sh
--- a/hack/lib/golang.sh	2022-11-09 13:28:30.000000000 +0000
+++ b/hack/lib/golang.sh	2022-11-15 10:59:34.414238448 +0000
@@ -482,7 +482,7 @@
   local go_version
   IFS=" " read -ra go_version <<< "$(GOFLAGS='' go version)"
   local minimum_go_version
-  minimum_go_version=go1.19
+  minimum_go_version=go1.18
   if [[ "${minimum_go_version}" != $(echo -e "${minimum_go_version}\n${go_version[2]}" | sort -s -t. -k 1,1 -k 2,2n -k 3,3n | head -n1) && "${go_version[2]}" != "devel" ]]; then
     kube::log::usage_from_stdin <<EOF
 Detected go version: ${go_version[*]}.
